### ISSUE TITLE
Shortcut for common User-Agent headers

### ISSUE THREAD

**Issue by gregbacchus** [jakubroztocil/httpie#757]
What would be useful for me is to have a shortcut for common User-Agent headers

e.g.

`http GET google.com --chrome`

**Comment by balalaXD** [comment]
yeah, it would be great if we have `Android`, `iPad`, `iPhone`, `phone`, `touch`, `desktop` etc

**Comment by t-mangoe** [comment]
I have tried this issue. To try it out, I added an option to specify the User-Agent for chrome.
<img width="887" alt="httpie1" src="https://user-images.githubusercontent.com/22494246/197307253-3f50cb44-62c1-4826-83ea-afa960594667.png">

In my implementation, this User-Agent String is defined as a constant. 
For this reason, it may be difficult to always display the latest User-Agent string.
<img width="778" alt="httpie2" src="https://user-images.githubusercontent.com/22494246/197307293-b81191aa-e2dc-429b-8b0e-3e26494e61fe.png">
